### PR TITLE
Fix for unicode errors when rendering problem html in OLI courses.

### PR DESCRIPTION
**Pull request by dcadams**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@dcadams Looks like relevant tests are failing:

`Exception: Superterrible error happened: \u2620`

File: common/lib/xmodule/xmodule/capa_base.py [hunk 1]
```diff
@@ -737,6 +737,13 @@ def get_problem_html(self, encapsulate=True, submit_notification=False):
 
         html = self.remove_tags_from_html(html)
 
+        # Short term fix for unicode errors when rendering problem
+        # html in OLI courses.
+        try:
+            html = html.decode('utf-8')
+        except UnicodeEncodeError:
```
Comment (stvstnfrd) [inline common/lib/xmodule/xmodule/capa_base.py]: What scenarios throw this exception?

**Comment by dcadams** [comment]
retest this please

**Comment by caesar2164** [comment]
[Hotseat: Probability & Statistics OLI /courseware page leads to unicode error](https://trello.com/c/4Uf8QXGn/1077-hotseat-probability-statistics-oli-courseware-page-leads-to-error)

**Comment by stvstnfrd** [comment]
@dcadams Can you attach a new test case to cover the new behavior?

**Comment by dcadams** [comment]
No longer necessary.

**Comment by stvstnfrd** [comment]
@dcadams What was the fix/workaround?

**Comment by dcadams** [comment]
Put this fix on privateprod which then enabled me to edit the offending capa probs in studio. The issue is with the way these particular problems substitute the problem text, and of course the contents of the problem text, into a var after the capa init.

**Pull request closed without merging**